swagger2.0/oic.sec.sdi.swagger.json

```diff
@@ -0,0 +1,171 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Security Domain Information",
+    "version": "v1.0-20191001",
+    "license": {
+      "name": "OCF Data Model License",
+      "url": "https://github.com/openconnectivityfoundation/core/blob/e28a9e0a92e17042ba3e83661e4c0fbce8bdc4ba/LICENSE.md",
+      "x-copyright": "copyright 2016-2017, 2019 Open Connectivity Foundation, Inc. All rights reserved."
+    },
+    "termsOfService": "https://openconnectivityfoundation.github.io/core/DISCLAIMER.md"
+  },
+  "schemes": ["http"],
+  "consumes": ["application/json"],
+  "produces": ["application/json"],
+  "paths": {
+    "/oic/sec/sdi" : {
+      "get": {
+        "description": "This Resource contains the information that identifies the OCF Security Domain to which the device belongs.\n",
+        "parameters": [
+          {"$ref": "#/parameters/interface"}
+        ],
+        "responses": {
+            "200": {
+              "description" : "Success",
+              "x-example":
+                {
+                  "rt": ["oic.r.sdi"],
+                  "uuid": "de305d54-75b4-431b-adb2-eb6b9e546014",
+                  "name": "Home",
+                  "priv": true
+                },
+              "schema": { "$ref": "#/definitions/Sdi" }
+            },
+            "400": {
+              "description" : "The request is invalid."
+            }
+        }
+      },
+      "post": {
+        "description": "Provision the OCF Security Domain information.\n",
+        "parameters": [
+          {"$ref": "#/parameters/interface"},
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": { "$ref": "#/definitions/Sdi-Update" },
+            "x-example":
+              {
+                  "uuid": "de305d54-75b4-431b-adb2-eb6b9e546014",
+                  "name": "Home",
+                  "priv": false
+              }
+          }
+        ],
+        "responses": {
+            "400": {
+              "description" : "The request is invalid."
+            },
+            "204": {
+              "description" : "The SDI is updated."
+            }
+        }
+      }
+    }
+  },
+  "parameters": {
+    "interface" : {
+      "in" : "query",
+      "name" : "if",
+      "type" : "string",
+      "enum" : [ "oic.if.baseline", "oic.if.rw" ]
+    }
+  },
+  "definitions": {
+    "Sdi" : {
+      "properties": {
+        "uuid": {
+          "description": "A UUID that identifies the Security Domain, set by DOTS during onboarding.",
```

Comment:
Normally, we just reference the definition of "uuid" in one of the Core schemas.  Can we not do that from a Security schema?

Reply:
Fixed. We do have a reference in AEL properties.